src/test/scala/com/microsoft/hyperspace/index/HybridScanSuite.scala

```diff
@@ -460,12 +461,77 @@ trait HybridScanSuite extends QueryTest with HyperspaceSuite {
           appendedFiles,
           deletedFiles,
           Seq(" <= 2000"))
+
+        // Check bucketSpec is not used.
+        val bucketSpec = planWithHybridScan collect {
+          case LogicalRelation(HadoopFsRelation(_, _, _, bucketSpec, _, _), _, _, _) =>
+            bucketSpec
+        }
+        assert(bucketSpec.length == 2)
+
+        // bucketSpec.head is for the index plan, bucketSpec.last is for the plan
+        // for appended files.
+        assert(bucketSpec.head.isEmpty && bucketSpec.last.isEmpty)
+
+        checkAnswer(baseQuery, filter)
+      }
+    }
+  }
+
+  test(
+    "Append-only: filter rule and non-parquet format," +
+      "appended data should be shuffled and merged by Union even with bucketSpec.") {
+    // Note: for delta lake, this test is also eligible as the dataset is partitioned.
```

Comment:
How should we handle this "Note"? This sounds like a TODO?

Reply:
It's note because this test is already included in delta lake hybrid scan tests.

Reply:
Shall we just remove the comment then? (not sure about the importance)